src/deepsparse/server/cli.py

```diff
@@ -156,6 +156,15 @@ def config(config_path: str, host: str, port: int, log_level: str):
     default="info",
     help="Sets the logging level.",
 )
+@click.option(
+    "--loggers",
+    type=str,
+    default="",
```

Comment:
```suggestion
    default=None,
```
And change loggers to Optional[str]?

Seems like this could remove the `loggers = loggers or None` line too

Reply:
right, had this originally, but click didn't seem to like the optionals, will take another look